### ISSUE TITLE
Option to silent %load_ext, or make %reload_ext work with %reset 

### ISSUE THREAD

**Issue by erihe251** [ipython/ipython#13145]
**Summary:**
I suggest adding a way of suppressing the `already loaded` warning of the `%load_ext` magic, such as an `-s` flag . 

The `%reload_ext` magic is not the same thing, and does not play well with the `%reset` magic. I don't know if this is intended or not.

---
**Details:**
If I run the following code
```python
%load_ext autoreload
%autoreload 2
%reset -f

import file1
```
several times, it gives me the warning `The autoreload extension is already loaded`, as expected.

If I try the following in order to avoid the warning 
```python
%reload_ext autoreload
%autoreload 2
%reset -f

import file1
```
what happens is that every time the autoreload is triggered, the code in `file1` is executed `n-1` times, where `n` is the number of times I have executed the above example.

I have not investigated enough to understand if this is intended behaviour or not, but I have searched online quite a bit for this issue and not found anything.

(Loading the extension once through a config file would of course be preferable, but I imagine it is not uncommon to be in situations where adding the magics at the top of your script file is the only reasonable option.)

**Comment by ignamv** [comment]
Yes, this message pollutes the notebook so I'm forced to filter it out before making a PDF. I think a natural place for the  `%load_ext` statements is at the top of the notebook along with the imports, and I think it would be reasonable for `%load_ext` to be idempotent like imports (do something on the first run, do nothing if re-run).

**Comment by simonkeys** [comment]
Using the `%%capture` cell magic:

```
%%capture
%load_ext autoreload
%autoreload 2
%reset -f
```

is working for me to suppress the warning.
